scripts/process_schemas.py

```diff
@@ -1,486 +0,0 @@
-"""
```

Comment:
Rather than remove this script, I think we should replace it with the equivalent that processes the protobuf schemas, keeping as much of the existing CLI as makes sense. (See comments on scripts/proto.py)